src/serac/physics/base_physics.cpp

```diff
@@ -188,7 +188,7 @@ void BasePhysics::initializeSummary(axom::sidre::DataStore& datastore, double t_
 
   // t: array of each time step value
   axom::sidre::View*         t_array_view = curves_group->createView("t");
-  axom::sidre::Array<double> ts(t_array_view, array_size);
+  axom::sidre::Array<double> ts(t_array_view, 0, array_size);
```

Comment:
I messed this up in my previous PR.  The constructor I missed that I was using was this:

```
array(sidre_view, num_elements, capacity=0);
```